src/SwedbankPay.Sdk.Infrastructure/SwedbankPayClient.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using SwedbankPay.Sdk.Consumers;
+using SwedbankPay.Sdk.Payments;
+using SwedbankPay.Sdk.PaymentOrders;
+
+namespace SwedbankPay.Sdk
+{
+    public class SwedbankPayClient : ISwedbankPayClient
+    {
+        private readonly HttpClient httpClient;
+
+        public SwedbankPayClient(HttpClient httpClient, IPaymentOrdersResource paymentOrders, IConsumersResource consumers, IPaymentsResource payments)
+        {
+            if (!ServicePointManager.SecurityProtocol.HasFlag(SecurityProtocolType.Tls12))
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)